stage1/gc/gc.go

```diff
@@ -85,3 +90,14 @@ func gcNetworking(podID *types.UUID) error {
 
 	return nil
 }
+
+func removeJournalLink(uuid *types.UUID) error {
+	machineID := strings.Replace(uuid.String(), "-", "", -1)
+	journalLink := filepath.Join("/var/log/journal", machineID)
```

Comment:
How about adding a note/link to the place where we define the name we use
to register

On Wed, Mar 16, 2016 at 5:39 PM Iago López Galeiras <
notifications@github.com> wrote:

> In stage1/gc/gc.go
> <https://github.com/coreos/rkt/pull/2289#discussion_r56367954>:
>
> > @@ -85,3 +90,14 @@ func gcNetworking(podID *types.UUID) error {
> >
> >  	return nil
> >  }
> > +
> > +func removeJournalLink(uuid *types.UUID) error {
> > +	machineID := strings.Replace(uuid.String(), "-", "", -1)
> > +	journalLink := filepath.Join("/var/log/journal", machineID)
>
> The name in /etc/machine-id is not the name we use to register the
> container with machined. The former is the pod UUID without dashes and the
> latter is rkt-${POD_UUID}.
>
> I'm pretty confident we'll keep the pod UUID without dashes as a machine
> id.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2289/files/287ee9150518e605cd022feae8fb3288c5e88bf1#r56367954>
>